app/assets/stylesheets/common/select-kit/toolbar-popup-menu-options.scss

```diff
@@ -7,6 +7,19 @@
         border-radius: 0;
       }
 
+      .select-kit-header {
+        height: 32px;
```

Comment:
I'm not sure we need to set the height here, the buttons next to it don't seem to have a height set.

Reply:
Yes it doesn’t but they have a 2px difference, given we use inside size to define size, any small difference can lead to height difference, others are buttons this one is select-kit. height should be fine here.